fem/restriction.cpp

```diff
@@ -618,15 +618,13 @@ ConformingFaceRestriction::ConformingFaceRestriction(
 {
    height = vdim*nf*face_dofs;
    width = fes.GetVSize();
-   if (!build) { return; }
    if (nf==0) { return; }
 
    CheckFESpace(f_ordering);
 
-   // Get the mapping from lexicographic DOF ordering to native ordering.
-   const FiniteElement *fe = fes.GetFE(0);
+   // Get the mapping from native DOF ordering to lexicographic ordering.
```

Comment:
It looks like you reverted my earlier fix in the comment. Was that unintentional?

Reply:
Yes, unintentional, sorry. I made some changes without first pulling. Feel free to re-change the comment.

Reply:
I reverted it

Reply:
Thanks!